web/src/app/workplace/components/task/task.component.ts

```diff
@@ -85,6 +96,19 @@ export class TaskComponent implements OnInit, OnDestroy {
     });
   }
 
+  cancelClaimTask() {
+    this.requestInProgress = true;
+    this.taskService
+      .cancelClaimTask(this.task.taskId)
+      .pipe(take(1))
+      .subscribe((task) => {
+        this.task = task;
+        this.taskService.publishUpdatedTask(task);
+        this.requestInProgress = false;
+      });
+    this.navigateBack();
+  }
+
   navigateBack() {
```

Comment:
has navigateBack still usages?

Reply:
yes, it is used to go back to the task details after cancelling a claim, completing a task or transferring a task.